scripts/perftests/RunPerformanceTestRunner.ps1

```diff
@@ -0,0 +1,47 @@
+Param(
```

Comment:
`RunPerformanceTestRunner` sounds weird, consider changing to `PerformanceTestRunner`